### DISCUSSION TITLE
Welcome to ethio-european-date-converter Discussions!

### DISCUSSION THREAD

**Discussion by melaku-z** [melaku-z/ethio-european-date-converter#315] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mintuka** [comment]
Hello melaku,
It is great work since it simplifies developer tasks.
But I am having problem when I try to use the package manager.
Example : 
            const ethDateTime = EthDateTime.fromEuropeanDate(new Date(2023,4,30,5,0))
            returned => 21/8/2015 , but it must be 22/8/2015
            Questions?
            How can I use the converter by adding hour and minute?

**Comment by melaku-z** [comment]
Thank you @Mintuka,
It looks like the problem is that `new Date(2023,4,30,5,0)` returns different date-time value based on in which timezone it was called. Therefore, `EthDateTime.fromEuropeanDate(new Date(2023,4,30,5,0))` also returns different ethiopian date.
For example, if the user (browser) is in timezone GMT+2, and the time for them is `5:00:00 , May 30`, then the Ethiopian date at that point is `22/8/2015`.
But if the user is in timezone GMT+11, and the time for them is `5:00:00 , May 30`, then the Ethiopian date at that point is `21/8/2015`.

So, one possible fix is that instead of `new Date(2023,4,30,5,0)` without specifying in which timezone the time is `5:00:00`, you have to create the js date with full information. Otherwise, you will get diffrent output based on where your timezone is.
